test/core/bin.js

```diff
@@ -132,7 +132,12 @@ _.each(fs.readdirSync(fixtureLoc), function (binName) {
       };
 
       var optionsLoc = testLoc + "/options.json"
-      if (fs.existsSync(optionsLoc)) _.merge(opts, require(optionsLoc));
```

Comment:
It'd require setting up sauce labs and setting up headless browser testing is usually painful.